src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -18,7 +18,9 @@ machine_type = "@MACHINETYPE@"
 
 # List of valid annotation names for the hypervisor
 # Each member of the list is a regular expression, which is the base name
-# of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
+# of the annotation
+# e.g. 1: "path" for "io.katacontainers.config.hypervisor.path"
+# e.g. 2: "pcie_lazy_attach_delay" for "io.katacontainers.config.hypervisor.pcie_lazy_attach_delay"
 enable_annotations = @DEFENABLEANNOTATIONS@
```

Comment:
First, it seems to me this enable annotations infrastructure belongs in a separate patch for easier review.

Second, I don't see how this is a regular expression in any sense.  Nor should it be.

Reply:
It has been deleted, I think I can add it to the document later.